songr/src/main/java/com/shaybrow/songr/HelloController.java

```diff
@@ -43,18 +50,20 @@ public String toUpperCase(
     public String albumsRender(
             Model m
     ){
-         ArrayList <Album> albums = new ArrayList<>();
-
-            albums.add(new Album("Humanz", "Gorillaz", 25, 2000,
-                    "https://img.apmcdn.org/72919f8057242b2533c1038f95667f66f4601d69/uncropped/1cdefb-20170323-humanz-art.jpg" ));
-            albums.add(new Album("The Moon & Antarctica", "Modest Mouse", 15, 1300,
-                    "https://images-na.ssl-images-amazon.com/images/I/61U1fpNPq2L._SX425_.jpg" ));
-            albums.add(new Album("Ugly is Beautiful", "Oliver Tree", 14, 1400,
-                    "https://t2.genius.com/unsafe/300x0/https%3A%2F%2Fimages.genius.com%2Fd01ae63bd1f7c9410e36dfe463f21c87.1000x1000x1.png" ));
+         List<Album> albums = albumRepo.findAll();
 
         m.addAttribute("albums", albums);
+
         return "albums.html";
     }
+    @PostMapping("/albums")

```

Comment:
You should create another controller for your albums! Don't do all of your work within a single controller.